docs/programming/tilebuffer.md

```diff
@@ -0,0 +1,285 @@
+# Multi-directional Scrolling (TileBuffer)
+
+## Basic description
+
+This tutoriel only focus on multi-directional scolling as used for RPG or games like Alien Breed, Chaos Engine or large platform game like Turrican or Superfrog.
+
+In order to do this multi-directional scolling, in ACE you need to use  the [managers/viewport/tilebuffer.h](../../include/ace/managers/viewport/tilebuffer.h). It manages for tiled background/map as a main entry.
+
+## Prerequisites
+
+The tile manager takes 2 pre-requisites :
+
+1. a set of tiles
+2. a map using this tiles.
+
+Last pre-requisites, you need the *palette* linked to your tiles graphic. [See Palette Tutorial](./palette.md)
+
+### Prepare the set of tiles
+
+Let's use this ressource from open game art : https://opengameart.org/content/overworld-grass-biome
+
+Get `TilesetGrass/overworld_tileset_grass.png` and convert it in 32 colors (use [grafx2](http://grafx2.chez.com/) for example).
```

Comment:
Since most of the tutorial concentrates on Aseprite, it would be good to give Aseprite instructions here too

Reply:
What do you think about this?